app/models/institution_program.rb

```diff
@@ -44,16 +44,23 @@ def va_bah
 
   # Finds exact-matching facility_code or partial-matching school and city names
   #
-  scope :search, lambda { |search_term|
+  scope :search, lambda { |search_term, fuzzy_search = false|
     return if search_term.blank?
 
     clause = [
       'institution_programs.facility_code = (:facility_code)',
-      'institutions.institution LIKE (:upper_search_term)',
-      'lower(description) LIKE (:search_term)',
-      'lower(institutions.physical_city) LIKE (:search_term)'
+      'lower(description) LIKE (:search_term)'
     ]
 
+    if fuzzy_search
+      clause << 'institution % :upper_search_term'
+      clause << 'physical_city % :upper_search_term'
```

Comment:
the `%` operator is most likely not going to work, see https://github.com/department-of-veterans-affairs/gibct-data-service/pull/685 for solution

Reply:
I can update that, was basing it off of what was there before #685

Reply:
@zurbergram pushed a change here to use `SIMILARITY` over the wildcards